chimney/src/main/scala/io/scalaland/chimney/DerivedTransformer.scala

```diff
@@ -49,4 +53,54 @@ object DerivedTransformer {
     (src: From, modifiers: Modifiers) =>
       toLG.from(genTransformer.transform(tag[From](fromLG.to(src)), modifiers))
   }
+
+  implicit def optionTransformer[From, To, Modifiers <: HList]
+    (implicit innerTransformer: DerivedTransformer[From, To, Modifiers])
+  : DerivedTransformer[Option[From], Option[To], Modifiers] =
+    (src: Option[From], modifiers: Modifiers) => src.map(innerTransformer.transform(_, modifiers))
+
+  implicit def someTransformer[From, To, Modifiers <: HList]
```

Comment:
I'm curious if single instance can be sufficient (for `X <: Option[T]`). Similarly for Either.